.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "compat-tests"]
```

Comment:
This might be good to call out in eng-team or engineering, just so people know that prefect has submodules now.

Reply:
Good call!